plugin/storage/es/options.go

```diff
@@ -157,30 +166,30 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		nsConfig.namespace+suffixTimeout,
 		nsConfig.Timeout,
 		"Timeout used for queries. A Timeout of zero means no timeout")
-	flagSet.Int64(
+	flagSet.Int(
```

Comment:
trying to remember why we changed this in the PR. I think since this PR started there were a few linter upgrades that created a lot of complaints with platform-specific `int`, so perhaps this change wasn't beneficial. With fixed-size ints you always know the range, regardless of the platform.

Reply:
@yurishkuro do we want to change the configuration to hold `int64` as well? there's a type mismatch otherwise